wiki/Building-on-Linux.md

```diff
@@ -7,27 +7,32 @@
 **Note**: these aren't the only things built at compile time, but merely what the end-user is expected to use.
 
 ## Preparing your system
+ This project depends upon
+   - cmake
+   - Qt (when building the gui)
+   - xorg-dev files
+   - libportal , libei (for wayland support)
+   - avahi
 
+
+## Example packages commands
+these can be use to install dependencies on various distros. They may not include what everything for the configuration you are trying to build so DO NOT blindly run these on your machine.
 ### Ubuntu/Debian/Raspbian
 
 ```shell
 sudo apt update && sudo apt upgrade
 sudo apt install git cmake make xorg-dev g++ libcurl4-openssl-dev \
                  libavahi-compat-libdnssd-dev libssl-dev libx11-dev \
-                 qttools5-dev qtbase5-dev libice-dev libsm-dev
+                 qttools6-dev qtbase6-dev libice-dev libsm-dev
```

Comment:
Regarding the Ubuntu/Debian/Raspbian equivalent packages for `qttools5-dev` and `qtbase5-dev`, seem as if the actual equivalent (at least when I was trying to install from source in my Raspbian system) is `qt6-tools-dev` and `qt6-base-dev`

Reply:
Thank you for the input , updated.